sabnzbd/postproc.py

```diff
@@ -517,6 +517,9 @@ def process_job(nzo: NzbObject):
                         nzo.set_unpack_info("Unpack", T("Failed to move files"))
                         all_ok = False
 
+            # Use par2 files to deobfuscate unpacked file names
+            newfiles = deobfuscate.recover_par2_names(newfiles)
```

Comment:
Let's add a Special setting to disable this, in case somehow it causes problems.